teammapper-backend/src/map/services/maps.service.spec.ts

```diff
@@ -119,8 +119,64 @@ describe('MapsController', () => {
   })
 
   describe('deleteOutdatedMaps', () => {
+    it('deletes a map based off of lastAccessed', async () => {
```

Comment:
typo: deletes a map based on `lastAccessed`. 

In addition: both values are way older than 30 days. I suggest to either lock the current date and set it to +1 day of lastAccessed or to make the dates relative, e.g. today - x days during creation.

Reply:
the date suggestion is true for basically all test, or am I missing something?

Reply:
e.g. https://stackoverflow.com/a/63377110

Reply:
Not sure it'd make sense for this test? If we lock the date to +1 of lastAccessed, the map wouldn't be deleted, since it's not older than 30 days. The other tests circumvent this by setting `new Date()` explicitly.

We could set each date explicitly using jest to make it more clear, but I don't think it would directly affect the tests.

Reply:
I think it would help a lot to lock the date to a specific date (see above link with jest) so it's easier to understand what the tests are actually testing. Otherwise we might accidentally test something completely different than we thought. 

And I am referring to all tests, not necessarily to this specific one. These are all tests for which the correct date is crucial.